gulpTasks/definition.js

```diff
@@ -65,7 +65,6 @@ function externalDefs() {
       './node_modules/@types/d3/index.d.ts',
       './lib/globalize/index.d.ts',
       './lib/jstimezonedetect/index.d.ts',
-      './lib/coveoanalytics/index.d.ts',
```

Comment:
I removed the coveoanalytics typedefinitions checked into our project to use the ones in the package directly.

This is technically breaking, since the defs in the package are more strict.
I also wasn't sure if we should continue to export the definitions?

Let me know if you prefer to:
a. Not touch the definitions/manually update what is missing in a non-breaking way (least risky)
b. Use the package definitions, and export these as part of the external defs.
c. Use the package definitions with exporting.
d. Other.